### DISCUSSION TITLE
Where is the Google Maps Web scrapper?

### DISCUSSION THREAD

**Discussion by Oudwins** [apify/apify-sdk-js#205] [Q&A]
### Which scraper is the feature request for?

puppeteer-scraper

### Feature

The code used to be hosted here on github but now its disappeared.

The current link in apify leads to a 404 error.

Apify page: https://apify.com/compass/crawler-google-places#resources-on-how-to-scrape-google-maps

404: https://github.com/drobnikj/crawler-google-places/blob/master/CHANGELOG.md

### Motivation

I want to be able to use & contribute to the scrapper

### Ideal solution or implementation, and any additional constraints

Make the code public again

**Comment by B4nan** [comment]
The repository is private.

**Comment by Oudwins** [comment]
@B4nan Yes I figured. May I ask why it was made private?

@Drormusic I messaged you a few weeks ago but no reply, maybe I got the email wrong?

**Reply by B4nan** [reply]
AFAIK it's one of our products we develop internally, we usually don't make such code public. I don't have a better answer, all I know is that it is intentional.
